<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>operateCamera</class>
 <widget class="QMainWindow" name="operateCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="title">
       <string>  摄像头参数</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>1666666</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>摄像头设备</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboCamera">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="checkStillImage">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>支持抓图</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkVideo">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>支持录像</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="LabDuration">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>录制时间：</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="chkMute">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>录像时静音</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="btnVideoFile">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>视频保存文件</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/images/images/104.bmp</normaloff>:/images/images/104.bmp</iconset>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editOutputFile">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>摄像头预览</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCameraViewfinder" name="viewFinder" native="true">
          <property name="minimumSize">
           <size>
            <width>450</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>抓取图片</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="LabCapturedImage">
          <property name="minimumSize">
           <size>
            <width>450</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>抓取的图片</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actStartCamera"/>
   <addaction name="actStopCamera"/>
   <addaction name="actVideoRecord"/>
   <addaction name="actVideoStop"/>
   <addaction name="actCapture"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actStartCamera">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/video_chat.ico</normaloff>:/images/video_chat.ico</iconset>
   </property>
   <property name="text">
    <string>开启摄像头</string>
   </property>
   <property name="toolTip">
    <string>开启摄像头</string>
   </property>
  </action>
  <action name="actStopCamera">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/remove_user.ico</normaloff>:/images/remove_user.ico</iconset>
   </property>
   <property name="text">
    <string>关闭摄像头</string>
   </property>
   <property name="toolTip">
    <string>关闭摄像头</string>
   </property>
  </action>
  <action name="actVideoRecord">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/5.ico</normaloff>:/images/5.ico</iconset>
   </property>
   <property name="text">
    <string>开始录像</string>
   </property>
   <property name="toolTip">
    <string>开始录像</string>
   </property>
  </action>
  <action name="actCapture">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/file_pictures.ico</normaloff>:/images/file_pictures.ico</iconset>
   </property>
   <property name="text">
    <string>抓图</string>
   </property>
   <property name="toolTip">
    <string>抓图</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/exit_24.ico</normaloff>:/images/exit_24.ico</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actVideoStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Shut Down.ico</normaloff>:/images/Shut Down.ico</iconset>
   </property>
   <property name="text">
    <string>停止录像</string>
   </property>
   <property name="toolTip">
    <string>停止录像</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header location="global">qcameraviewfinder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>operateCamera</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
